content/blogs/ecosystem.md

```diff
@@ -1,9 +1,6 @@
 ---
 title: "Guide to Berkeley Ecosystem"
 date: 2021-02-19T11:05:52-05:00
-author: "BTF Team"
```

Comment:
QQ: I like this change but this doesn't disallow us from using "author" tag in the future, right? Might have some alum write up some stuff (Johnathan, Alex Kern, etc) so might want it for that purpose

Reply:
@itsmingjie bumping! Lmk if Github async is good workflow to make web stuff less friction, too, btw -- happy to work on whatever you're accustomed to :)